### PR TITLE
Docs: Added summary to math and pageview endpoints

### PR DESCRIPTION
**Pull request by Pchelolo**
The summary is a short description of the API shown on the right side of collapsed entry point doc in the [swagger docs](https://en.wikipedia.org/api/rest_v1/?doc=). We have them everywhere except mathoid and pageviews.

cc @wikimedia/services @milimetric

### CODE REVIEW

File: v1/pageviews.yaml [hunk 1]
```diff
@@ -181,6 +184,7 @@ paths:
     get:
       tags:
         - Pageviews data
+      summary: Get most viewed articles for a project.
```
Comment (gwicke) [inline v1/pageviews.yaml]: Minor: "Get the most viewed articles for a project." reads slightly more natural.

**Comment by gwicke** [comment]
LGTM overall.

File: v1/mathoid.yaml [hunk 2]
```diff
@@ -94,6 +95,7 @@ paths:
   /math/render/{format}/{hash}:
     get:
       tags: ['Math']
+      summary: Get rendered formula in a given format.
```
Comment (d00rman) [inline v1/mathoid.yaml]: `s/a/the/`

Comment (Pchelolo) [inline v1/mathoid.yaml]: @d00rman done.

**Pull request merged**